tests/server/auth/test_client.py

```diff
@@ -0,0 +1,156 @@
+import pytest
+from unittest import mock
+
+import mlflow
+from mlflow.server.auth.client import AuthServiceClient
+from mlflow.server.auth.routes import (
+    CREATE_EXPERIMENT_PERMISSION,
+    GET_EXPERIMENT_PERMISSION,
+    UPDATE_EXPERIMENT_PERMISSION,
+    DELETE_EXPERIMENT_PERMISSION,
+    CREATE_REGISTERED_MODEL_PERMISSION,
+    GET_REGISTERED_MODEL_PERMISSION,
+    UPDATE_REGISTERED_MODEL_PERMISSION,
+    DELETE_REGISTERED_MODEL_PERMISSION,
+)
+from tests.helper_functions import random_str
+
+
+@pytest.fixture
+def client():
+    yield AuthServiceClient("http://tracking_uri")
```

Comment:
```suggestion
    return AuthServiceClient("http://tracking_uri")
```

since there is no teardown.